django_ninja_crudl/renderers.py

```
@@ -0,0 +1,30 @@
+import json
```

Ruff (0.11.9)

Missing docstring in public module

(D100)